packages/cli/src/eventbus/message-event-bus-destination/message-event-bus-destination-webhook.ee.ts

```diff
@@ -95,7 +105,63 @@ export class MessageEventBusDestinationWebhook
 		if (options.sendPayload) this.sendPayload = options.sendPayload;
 		if (options.options) this.options = options.options;
 
+		const axiosSetting = this.buildAxiosSetting(options);
+
+		this.axiosInstance = axios.create(axiosSetting);
+
 		this.logger.debug(`MessageEventBusDestinationWebhook with id ${this.getId()} initialized`);
+		this.logger.debug(
+			`MessageEventBusDestinationWebhook with options ${JSON.stringify(this.options)}`,
+		);
+	}
+
+	private buildAxiosSetting(
+		options: MessageEventBusDestinationWebhookOptions,
+	): CreateAxiosDefaults {
+		const axiosSetting: CreateAxiosDefaults = {
+			headers: {},
+			method: options.method as Method,
+			url: options.url,
+			maxRedirects: 0,
+		} as AxiosRequestConfig;
+
+		if (options.options?.redirect?.followRedirects) {
```

Comment:
I would just rename the options arg as "axiosParameters" or something, to prevent options.options